pkg/kubelet/kubelet.go

```diff
@@ -1742,7 +1743,7 @@ func (kl *Kubelet) syncPod(ctx context.Context, updateType kubetypes.SyncPodType
 		for _, r := range result.SyncResults {
 			if r.Error != kubecontainer.ErrCrashLoopBackOff && r.Error != images.ErrImagePullBackOff {
 				// Do not record an event here, as we keep all event logging for sync pod failures
-				// local to container runtime so we get better errors
```

Comment:
Add period at the end to make it become a complete sentence.
`local to container runtime, so we get better errors.`

Reply:
ok